app/services/scrape.py

```diff
@@ -157,3 +158,13 @@ def get_conversation(self, conversation_id: TwitterID) -> List[tweepy.Tweet]:
                 tweet_fields="referenced_tweets",
             ).flatten()
         )
+
+    def get_full_details(self, user_id: TwitterID, tweets_num: int):
+        user = self.get_user_by_id(user_id)
+        recent_tweets = self.api.user_timeline(user_id=user_id, count=tweets_num)
+        tweets_dict_list = []
+        for tweet in recent_tweets:
+            tweet._json.pop("user")
```

Comment:
@dirtyg2120 Don't use `_json`

Reply:
@phhai-hcmut if not use `_json`, so i will have to implement the whole Schema for it right? u forgot to answer me this on slack :))

Reply:
Just reuse `User` and `Tweet` from `app/models/twitter.py`